packages/smart-wallet/test/test-psm-integration.js

```diff
@@ -155,108 +156,148 @@ test('want stable', async t => {
   t.is(purseBalance(computedState, stableBrand), swapSize - 1n);
 });
 
-// TODO will be be fixed in #6110
-test.skip('govern offerFilter', async t => {
+const INVITATION_MAKERS_TEXT = 'PSM charter member invitation';
```

Comment:
consider "_DESC" since it's the description value

Reply:
also consider exporting, since this is part of the API for looking up invitations in purses